test/performance/odp_queue_perf.c

```diff
@@ -443,13 +481,79 @@ static int run_test(void *arg)
 	stat->deq_retry = num_deq_retry;
 	stat->enq_retry = num_enq_retry;
 
-	return ret;
+	return 0;
+}
+
+static void map_queues_to_threads(test_global_t *global)
+{
+	test_options_t *opt = &global->options;
+
+	if (!opt->private_queues) {
+		for (uint32_t i = 0; i < opt->num_queue; i++) {
+			for (uint32_t j = 0; j < opt->num_cpu; j++) {
+				global->thread_args[j].src_queue_id[i] = i;
```

Comment:
Easier to understand if `global->thread_args[]` elements were accessed via a helper pointer (also below).

Reply:
Changed in V2.